src/display.rs

```diff
@@ -143,12 +146,20 @@ pub fn draw_it(
     is_reversed: bool,
     no_colors: bool,
     no_percents: bool,
+    by_filecount: bool,
     root_node: Node,
 ) {
     if !permissions {
         eprintln!("Did not have permissions for all directories");
     }
-    let terminal_width = (get_width_of_terminal() - 14) as usize;
+    let num_chars_needed_on_left_most = if by_filecount {
+        let max_size = root_node.children.iter().map(|n| n.size).fold(0, max);
+        max_size.separate_with_commas().chars().count()
+    } else {
+        5 // Under normal usage we need 5 chars to display the size of a directory
+    };
+
+    let terminal_width = get_width_of_terminal() as usize - 9 - num_chars_needed_on_left_most;
```

Comment:
Shouldn't this be 9 + num_chars_needed_on_left_most?

Reply:
I think it depends how you read it
```(terminal_width - 9 ) - nums_chars_need```
vs
```terminal_width - (9 + nums_chars_need)```

Reply:
Oh, right